### PR TITLE
[iOS] Fix entry renderer font size

### PR DESCRIPTION
**Pull request by kingces95**
### Description of Change ###

Fix entry renderer font size

### Bugs Fixed ###

- https://bugzilla.xamarin.com/show_bug.cgi?id=59925

### API Changes ###

None

### Behavioral Changes ###

None

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

**Comment by kingces95** [comment]
entryfontsize

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 123456, "Issue Description", PlatformAffected.Default)]
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs]: Don't forget to fix the test case attribute and class/file name

File: Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs [hunk 2]
```diff
@@ -12,21 +14,23 @@ public class EntryRenderer : ViewRenderer<Entry, UITextField>
 		UIColor _defaultTextColor;
 		bool _disposed;
 
-		public EntryRenderer()
+		static readonly int basetHeight = 30;
```
Comment (samhouts) [inline Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs]: typo?

File: Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs [hunk 3]
```diff
@@ -12,21 +14,23 @@ public class EntryRenderer : ViewRenderer<Entry, UITextField>
...
-			//https://stackoverflow.com/a/36569247/1063783
-			//we get the current value, and restor it, to allow custom renderers to change the border style
-			var borderStyle = Control.BorderStyle;
-			Control.BorderStyle = UITextBorderStyle.None;
-			var size = Control.GetSizeRequest(widthConstraint, double.PositiveInfinity);
-			Control.BorderStyle = borderStyle;
-			return size;
+			var baseResult = base.GetDesiredSize(widthConstraint, heightConstraint);
+
+			NSString testString = new NSString("Tj");
```
Comment (samhouts) [inline Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs]: Like...Tijuana?

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+				}
+			};
+
+			// Initialize ui here instead of ctor
+			Content = stack;
+		}
+
+#if UITEST
+		[Test]
+		public void Issue123456Test ()
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs]: And don't forget to fix the test name

File: Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs (hunk 3 above)
Comment (jassmith) [reply Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs]: nah just using a string that has both a cap and a descender

**Comment by kingces95** [comment]
@samhouts Thanks for all the comments. Will address should @jassmith ask me to continue to shuttle this issue along...

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 59925, "Issue Description", PlatformAffected.Default)]
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs]: Not a blocker, but we might want that issue description fixed.

Comment (jassmith) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla123456.cs]: Wtf I know I changed that...

**Comment by jassmith** [comment]
Failures are not the result of this PR and seem to be random (at a glance)

**Pull request merged**